openquakeplatform/openquakeplatform/icebox/models.py

```diff
@@ -72,14 +72,15 @@ def process_layers(self):
         # we create a database View per each output, then we publish the
         # db view on geoserver as a postgis vector layer. Finally, we
         # let geonode slurp the geoserver layer
-        for olayer in self.outputlayer_set.all():
```

Comment:
I need to skip `self.create_geonode_map(layers)`, which is outside the loop, when `len(self.outputlayer_set.all())` is less then 1.
There's no reason to try the loop is there are no layers, and at the same time I skip the map creation.